dagshub/data_engine/model/datasource.py

```diff
@@ -304,26 +308,44 @@ def visualize(self, **kwargs) -> "fo.Session":
     def fields(self) -> List[MetadataFieldSchema]:
         return self.source.metadata_fields
 
+    def annotate(self) -> Optional[str]:
+        """
+        Sends all datapoints in the datasource for annotation in Label Studio.
+        It's recommended to not send a huge amount of datapoints to be annotated at once, to avoid overloading
```

Comment:
Not sure why this is here. If this isn't the recommended way to send a lot of annotations, then we should direct people to the correct way, or fix it so that it works. As it is, this sounds bad